helper_crates/vtable/Cargo.toml

```diff
@@ -13,3 +13,4 @@ homepage = "https://sixtyfps.io"
 [dependencies]
 vtable-macro = { version = "=0.1.6", path = "./macro" }
 const-field-offset = { version = "0.1", path = "../const-field-offset" }
+atomic-polyfill = "0.1.5"
```

Comment:
i wonder if it should be a opetional dependency

Reply:
No strong feeling here - although my guts feeling is that less "#ifdefs" are easier to maintain. Happy to change it to a #cfg if you prefer - let me know :)

Reply:
you're right, it is fine to keep it because on other platform it is a no-op anyway